vincechain/assetlist.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "vincechain",
+  "assets": [
+    {
+      "description": "Vince Chain is an Industry-Grade Blockchain For DeFi, Remote work, Gaming, Web3 and will power the new internet.",
+      "denom_units": [
+        {
+          "denom": "avce",
+          "exponent": 0
+        },
+        {
+          "denom": "vce",
+          "exponent": 18
+        }
+      ],
+      "base": "avce",
+      "name": "VinceChain",
+      "display": "vincechain",
+      "symbol": "VCE",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/vincechain/images/vince.png"
```

Comment:
the image uploaded is named 'logo', not 'vince'. Please correct one or the other.